app/controllers/api/v1/votes_controller.rb

```diff
@@ -22,6 +22,14 @@ def destroy
     end
   end
 
+  def index
```

Comment:
if you want the url to look like that and not mess with Rails default url schemas you have to call it index. It's just not what we normally do. We normally do:

`/tasks/find?task_id=123` if we are finding something based on a unique attribute that is not the `id`.